webdev/test/serve/chrome_test.dart

```
@@ -25,6 +27,6 @@ void main() {
 
   test('debugger is working', () async {
     var tabs = await chrome.chromeConnection.getTabs();
-    expect(tabs.length, equals(1));
+    expect(tabs.firstWhere((tab) => tab.url == _googleUrl), isNotNull);
```

If this was not found it would throw instead of return null, I think the way this expect is written is misleading (though the test would still fail).

How about something like this:

```dart
expect(tabs, contains(TypeMatcher<ChromeTab>().having((t) => t.url, 'url', _googleUrl)));
```

---

The other benefit of this approach is that we'd get #148 for free. The `toString()` on `tabs` will already include the urls for all the tabs. Even if we correct this version to use an `orElse` argument, the failure message would not be helpful.

---

https://github.com/dart-lang/webdev/pull/154